guides/common/modules/con_performing-post-upgrade-tasks.adoc

```diff
@@ -4,6 +4,9 @@
 Some of the procedures in this section are optional.
 You can choose to perform only those procedures that are relevant to your installation.
 
+If you use custom templates created from a cloned default template, create the custom template again after the upgrade.
+Sometimes the default templates are changed during the upgrade, this will ensure that the custom templates are compatible with the latest version.
```

Comment:
```suggestion
The default templates are changed during the upgrade. 
This will ensure that the custom templates are compatible with the latest version.
```
Following on from Malhar's suggestion, I think this should be two sentences.